app/components/c-product-detail.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <div :class="b()">
+
+    <div :class="b('inner')">
+
+      <section :class="b('product')"> <pre>{{ product }}</pre></section>
+
+      <div :class="b('content')">
+
+        <div :class="b('main')">
+          <section :class="b('details')"> details</section>
+          <section :class="b('related')"> related</section>
+          <section :class="b('accessories')"> accessories</section>
+        </div>
+
+        <div :class="b('sidebar')">
+          <aside :class="b('recommendation')"> recommendation</aside>
+        </div>
+
+      </div>
+
+    </div>
+
+  </div>
+
+</template>
+
+<script>
+  import { mapGetters } from 'vuex';
+
+  export default {
+    name: 'c-product-detail',
+
+    // mixins: [],
+
+    // props: {},
+    // data() {
+    //   return {};
+    // },
+
+    computed: {
+      ...mapGetters({
+        product: 'product/getProduct'
+      })
+
+    }
+    // watch: {},
+
+    // beforeCreate() {},
+    // created() {},
+    // beforeMount() {},
+    // mounted() {},
+    // beforeUpdate() {},
+    // updated() {},
+    // activated() {},
+    // deactivated() {},
+    // beforeDestroy() {},
+    // destroyed() {},
+
+    // methods: {},
+    // render() {},
+  };
+</script>
+
+<style lang="scss">
+  .c-product-detail {
+    &__product {
+      width: 100%;
+    }
+
+    &__content {
+      @include media(sm) {
+        display: flex;
+      }
+    }
+
+    &__main {
+      @include media(sm) {
+        flex: 1;
+      }
+    }
+
+    &__sidebar {
+      @include media(sm) {
+        flex-basis: 25%;
```

Comment:
- [ ] Please use `flex-basis: percentage(3 / 12)` to make it clear this is 3 column width.

Reply:
Do you think it would make sense to ad a `col(<amountOfCols>)` function so we could say `width: col(3);`?

Reply:
I think, that this might be a little 'over engineered, since it just returns a percentage value and there is not much code to write. Though, the advantage would be, that we can set the amount of columns (grid) from a central point e.g. `$grid-cols:12`. What do you think? Shall I create the mixin or leave it?

Reply:
Ok, then lets stay with `percentage()` for now. This I would definitely use because there are some rations where we don't want to write the percent value ourself.

Reply:
@patric-eberle Didn't know this function, thank you.